backend/server/static/api/meetingbook.yaml

```
@@ -163,22 +164,230 @@ paths:
         "401":
           description: "Unauthorized"
       security:
-        - basic_auth: []    
-  /bookings/{booking_id}:
+        - basic_auth: []
+  /booking_settings:
+    summary: endpoint for the settings of the booking, controlled by admin.
+    get:
+      summary: get current settings
```

It's needed to rewrite "summary" into description and add this line in "get" after "summary"

```suggestion
    get:
      summary: get current settings
     description: get current settings of the booking, controlled by admi
```

---

this summary for all /booking_settings